locales/ga_IE/LC_MESSAGES/duckduckgo.po

```diff
@@ -4999,6 +4999,9 @@ msgstr "Físeáin"
 msgid "Vietnam"
 msgstr "Vítneam"
 
+msgid "View File"
+msgstr "Féach ar an gcomhad"
+
```

Comment:
Oops, looks like I might have duplicated this when resolving the conflicts. Fixing now.

```suggestion
```